massa-protocol-worker/src/connectivity.rs

```diff
@@ -283,9 +279,11 @@ pub(crate) fn start_connectivity_thread(
                                             continue;
                                         }
```

Comment:
we can remove unwrap() on next()

Reply:
Removed using the change I described below